scale/ingest/test/strike/monitors/test_dir_monitor.py

```diff
@@ -18,7 +18,7 @@ def test_validate_configuration_missing_transfer_suffix(self):
         config = {
             'type': 'dir-watcher'
         }
-        self.assertRaises(InvalidMonitorConfiguration, DirWatcherMonitor().validate_configuration, config)
+        DirWatcherMonitor().validate_configuration(config)
```

Comment:
If we don't care about transfer_suffix being there or not, might as well just remove this test.